programs/tensorflowDetection2D/MainDetector.cpp

```diff
@@ -116,24 +124,24 @@ int maindetector::detect(std::string labels, std::string graph, std::string vide
     double fps = 0.;
     time_t start, end;
     time(&start);
-
-    // Adquiere imagen de la fuente de video
-    cv::VideoCapture cap(source_video);
-
     tensorflow::TensorShape shape = tensorflow::TensorShape();
     shape.AddDim(1);
-    shape.AddDim((tensorflow::int64)cap.get(cv::CAP_PROP_FRAME_HEIGHT));
-    shape.AddDim((tensorflow::int64)cap.get(cv::CAP_PROP_FRAME_WIDTH));
+    shape.AddDim((tensorflow::int64)800);//cap.get(cv::CAP_PROP_FRAME_HEIGHT)->800
+    shape.AddDim((tensorflow::int64)600);//cap.get(cv::CAP_PROP_FRAME_WIDTH)->600
```

Comment:
In other implementations, I believe we get this from the received image. Can you check this out?

Reply:
Yes, i got  width and height parameters with OpenCV, because `cap` was `VideoCapture` object. I´m trying to use YARP to get the parameters from `IlpImage` directly and remove `cap.get..`.

Reply:
Something like 05a7388ee5eb33a074c866fa4a5a7de57094318d , that gets width and height from yarp source without OpenCV?

Reply:
> Something like [05a7388](https://github.com/roboticslab-uc3m/vision/commit/05a7388ee5eb33a074c866fa4a5a7de57094318d) , that gets width and height from yarp source without OpenCV?

Yes, precisely.